tests/suites/test_suite_mpi.function

```diff
@@ -967,58 +954,56 @@ void mpi_mod_raw_cond_swap( data_t * input_X,
     mbedtls_mpi_mod_modulus m;
     size_t limbs_X = CHARS_TO_LIMBS( input_X->len );
     size_t limbs_Y = CHARS_TO_LIMBS( input_Y->len );
-    size_t len_X = limbs_X * sizeof( mbedtls_mpi_uint );
-    size_t len_Y = limbs_Y * sizeof( mbedtls_mpi_uint );
-    size_t len_m = len_X;
+    size_t limbs = limbs_X;
+    size_t len = limbs * sizeof( mbedtls_mpi_uint );
 
-    TEST_ASSERT( limbs_X <= MAX_LEN );
-    TEST_ASSERT( limbs_Y <= MAX_LEN );
+    TEST_ASSERT( limbs_X == limbs_Y );
```

Comment:
Non-blocker: `TEST_EQUAL( limbs_X, limbs_Y )` gives better feedback on failure. (Here and in a few other places.)